daml-lf/language/src/main/scala/com/digitalasset/daml/lf/language/PackageInterface.scala

```diff
@@ -418,6 +418,28 @@ private[lf] class PackageInterface(signatures: PartialFunction[PackageId, Packag
   val packageLanguageVersion: PartialFunction[PackageId, LanguageVersion] =
     signatures andThen (_.languageVersion)
 
+  def interfaceImplementations(
+      pkgId: PackageId,
```

Comment:
If I understand this correctly, the property we rely on here is that to get the interface implementations for a set of packages, we can call this function for each package id and then combine the result via map/set union. Maybe useful to spell that out.

Reply:
not applicable anymore